packages/contracts/contracts/test/MajorityVotingMock.sol

```diff
@@ -7,16 +7,11 @@ import "../voting/majority/MajorityVotingBase.sol";
 contract MajorityVotingMock is MajorityVotingBase {
     function initializeMock(
         IDAO _dao,
-        uint64 _totalSupportThresholdPct,
-        uint64 _relativeSupportThresholdPct,
+        uint64 _supportThreshold,
+        uint64 _minParticipation,
         uint64 _minDuration
```

Comment:
Are we currently explaining the difference between threshold and min in the docs? If we don't, feel free to create a task to track this in the future

Reply:
However, added an explanation in 04a339cda63bc7a6e1d59905f61330be41dcf162.